src/admin/adminActions.ts

```diff
@@ -31,14 +40,25 @@ export const editEvent: ThunkActionCreator = (event) => async (dispatch) => {
   return new Promise(async (resolve, reject) => {
     try {
       const url = `${Config.API_URL + Config.routes.events.event}/${event.uuid}`;
-      await fetchService(url, 'PATCH', 'json', {
+      const data = await fetchService(url, 'PATCH', 'json', {
         requiresAuthorization: true,
         payload: JSON.stringify({
           event,
         }),
         onFailCallback: () => dispatch(logoutUser()),
       });
 
+      if (typeof event.cover === 'object') {
+        const formdata = new FormData();
+        formdata.append('image', event.cover);
+
+        const url2 = `${Config.API_URL + Config.routes.events.picture}/${data.event.uuid}`;
```

Comment:
Similar nitpick as before:
```suggestion
        const imageUploadRoute = `${Config.API_URL + Config.routes.events.picture}/${data.event.uuid}`;
```